javalib/src/main/scala/java/util/AbstractSequentialList.scala

```diff
@@ -0,0 +1,45 @@
+package java.util
+
+abstract class AbstractSequentialList[E] protected ()
+    extends AbstractList[E] {
+
+  def get(index: Int): E = {
+    val iter = listIterator(index)
+    if (iter.hasNext) iter.next()
+    else throw new IndexOutOfBoundsException(index.toString)
+  }
+
+  override def set(index: Int, element: E): E = {
+    val iter = listIterator(index)
+    val ret = iter.next()
+    iter.set(element)
+    ret
+  }
+
+  override def add(index: Int, element: E): Unit =
+    listIterator(index).add(element)
+
+  override def remove(index: Int): E = {
+    val iter = listIterator(index)
+    val ret = iter.next()
```

Comment:
It will fail for `index == size` with a `NoSuchElementException` instead of `IndexOutOfBounds`. Also, add the `()` to the `remove`.
It should probably be:
```
val iter = listIterator(index)
if (!iter.hasNext)
  throw new IndexOutOfBoundsException
val ret = iter.next()
iter.remove()
ret